spec/formatters_spec.lua

```diff
@@ -170,5 +189,8 @@ describe("Test Formatters", function()
     assert.spy(s).was.called(1)
     assert:remove_formatter(s)
   end)
-
 end)
```

Comment:
can you please remove unrelated changes like this? It clutters the diff and makes it hard(er) to review

Reply:
Gone.